File: src/Core.Domain/Entities/Servicio.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TurnosApp.Core.Domain.Entities;

using TurnosApp.Core.Domain.Common;

public class Servicio : TenantEntity
{
    public string Nombre { get; set; } = string.Empty;          // ej: "Consulta", "Radiografía"
    public string? Descripcion { get; set; }
    public int DuracionMinutos { get; set; }
    public decimal Precio { get; set; }
    public bool Activo { get; set; } = true;

    // Navegación (M:N hacia Turno vía tabla de unión explícita)
    public ICollection<TurnoServicio> TurnoServicios { get; set; } = [];
}

File: turnos-app-frontend/src/types/Servicio.ts
// src/types/Servicio.ts
export interface Servicio {
  id: number;
  nombre: string;
  descripcion: string;
  duracionMinutos: number;
  precio: number;
}

export interface ServicioCreateDTO {
  nombre: string;
  descripcion: string;
  duracionMinutos: number;
  precio: number;
  
}

export interface ServicioUpdateDTO {
  nombre: string;
  descripcion: string;
  duracionMinutos: number;
  precio: number;
  
}